src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -1081,22 +1064,53 @@ describe("AuthenticationController#logout", () => {
     const response = await controller.logout(req);
     response.apply(res);
 
-    expect(controller).toBeTruthy();
+    expect(mockGetLollipop).not.toHaveBeenCalled();
+    expect(mockDelLollipop).not.toHaveBeenCalled();
+    expect(mockRevokePreviousAssertionRef).not.toHaveBeenCalled();
     expect(mockDel).not.toBeCalled();
     expect(res.status).toHaveBeenCalledWith(400);
     expect(res.json).toHaveBeenCalledWith(badRequestErrorResponse);
   });
 
-  it("should fail if the session can not be destroyed", async () => {
+  it("should fail if the assertionRef can not be deleted for a redis error", async () => {
     const res = mockRes();
     const req = mockReq();
     req.user = mockedUser;
-    mockDel.mockReturnValue(Promise.resolve(E.right(false)));
 
+    mockDelLollipop.mockResolvedValueOnce(E.left(new Error("Redis error")));
+    mockDel.mockResolvedValueOnce(E.right(false));
+
+    expect(controller).toBeTruthy();
```

Comment:
this line now can be deleted.
```suggestion
```

Reply:
https://github.com/pagopa/io-backend/pull/985/commits/f4a0b10c05d56dbe1711f2d258af6c47eabd2c8b